plugins/modules/gcp_cloudbuild_trigger.py

```diff
@@ -679,10 +691,7 @@ def main():
 
     state = module.params['state']
 
-    if module.params['id']:
-        fetch = fetch_resource(module, self_link(module))
-    else:
-        fetch = {}
+    fetch = fetch_resource(module, self_link(module))
```

Comment:
@rambleraptor should I be concerned with this change?

Reply:
This is fine!